src/v2/guide/transitioning-state.md

```diff
@@ -255,9 +255,9 @@ new Vue({
 </style>
 {% endraw %}
 
-## Dynamic State Transitions
+## Transitions d'état dynamiques
 
-Just as with Vue's transition components, the data backing state transitions can be updated in real time, which is especially useful for prototyping! Even using a simple SVG polygon, you can achieve many effects that would be difficult to conceive of until you've played with the variables a little.
+Tout comme pour les composants de transition de Vue, les données de la sauvegarde des transitions de l'état peuvent être mises à jour en temps réel, ce qui est particulièrement utile pour le prototypage ! Même en utilisant un polygone SVG simple, vous pouvez obtenir de nombreux effets qui seraient difficile à concevoir tant que vous n'avez pas un peu joué avec les variables.
```

Comment:
the data backing state transitions => pas évident celui-là... perso je dirais "les transitions d'état de données" ; généralement on parle de sauvegarde quand on utilise le verbe "back" avec la préposition "up", mais sans cette préposition je ne pense pas que ce soit ça, d'autant que je ne vois pas où une sauvegarde se fait techniquement. Ici je pense que par "backing" il entend plus un rôle d'assistanat, de soutien en tâche de fond, comme quoi ces transitions sont gérées par Vue en arrière-plan sans qu'on ait à s'en soucier.